mnelab/model.py

```diff
@@ -583,7 +585,8 @@ def get_compatibles(self):
                 continue
             if d["data"].info["bads"] != data.info["bads"]:
                 continue
-            if not all(d["data"]._cals == data._cals):
+            if d["dtype"] == "raw" and \
+                not all(d["data"]._cals == data._cals):
```

Comment:
`if d["dtype"] == "raw" and any(d["data"]._cals != data._cals):` (also, please try to avoid breaking lines with `\`)